packages/common/src/permissions/_internal/checkAssertion.ts

```diff
@@ -48,6 +48,12 @@ export function checkAssertion(
       case "neq":
         response = equalityAssertions(assertion, propValue, val);
         break;
+      case "starts-with":
+      case "ends-with":
+      case "not-starts-with":
+      case "not-ends-with":
+        response = stringAssertions(assertion, propValue, val);
+        break;
```

Comment:
I believe there was no other way to do this. Since the value of hubUrl will contain a `:`, the `neq` assertion type understands it to not be a literal value but an instruction to lookup another value on the entity/context hash.